features/aanschrijfwijze/hoffelijkheidstitel.feature

```diff
@@ -0,0 +1,330 @@
+    - het geslacht van de persoon is "M" (man)
+    - er is geen vrouwelijke vorm voor de adellijke titel van de partner (zie tabel hierboven)
+
+    Hierbij geldt verder dat:
+    - spaties aan het begin of einde van de aanschrijfwijze.naam worden verwijderd
+    - dubbele spaties uit de aanschrijfwijze.naam worden verwijderd
+    - een spatie vóór of na de "-" uit de aanschrijfwijze.naam worden verwijderd
+    - de voorletters worden opgenomen zoals beschreven in voorletters.feature
+    - de hoffelijkheidstitel wordt in kleine letters geschreven
+    - de adellijke titel wordt in kleine letters geschreven
+    - de eerste naamcomponent in de aanschrijfwijze.naam begint met een hoofdletter
+
+    Abstract Scenario: Vrouw met adellijke titel en partner met adellijke titel "<titel partner>"
+      Gegeven de persoon met burgerservicenummer '000000243' heeft de volgende gegevens
+      | naam                        | waarde |
+      | geslachtsaanduiding (04.10) | V      |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde      |
+      | voornamen (02.10)                    | Anne-Fleur  |
+      | adellijke titel of predicaat (02.20) | BS          |
+      | voorvoegsel (02.30)                  | de          |
+      | geslachtsnaam (02.40)                | Bonne Fooie |
+      | aanduiding naamgebruik (61.10)       | N           |
+      En de persoon heeft een 'partner' met de volgende 'naam' gegevens
+      | naam                                 | waarde          |
+      | adellijke titel of predicaat (02.20) | <titel partner> |
+      | voorvoegsel (02.30)                  | van den         |
+      | geslachtsnaam (02.40)                | Aedel           |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                           |
+      | type                | RaadpleegMetBurgerservicenummer  |
+      | burgerservicenummer | 000000243                        |
+      | fields              | adressering.aanschrijfwijze.naam |
+      Dan heeft de response een persoon met alleen de volgende 'adressering' gegevens
+      | naam                          | waarde                    |
+      | aanschrijfwijze.naam          | <naam in aanschrijfwijze> |
+      | aanschrijfwijze.aanspreekvorm | <aanspreekvorm>           |
+
+      Voorbeelden:
+      | titel partner | naam in aanschrijfwijze                           | aanspreekvorm            | omschrijving                          |
+      | B             | A. barones de Bonne Fooie-barones van den Aedel   | De hoogwelgeboren vrouwe | aanspreekvorm uit hoffelijkheidstitel |
+      | BS            | A. barones de Bonne Fooie-van den Aedel           | De hoogwelgeboren vrouwe | aanspreekvorm uit eigen titel         |
+      | G             | A. barones de Bonne Fooie-gravin van den Aedel    | De hoogwelgeboren vrouwe | aanspreekvorm uit hoffelijkheidstitel |
```

Comment:
als aanspreekvorm uit hoffelijkheidstitel moet komen, dan zou volgens mij hier aanspreekvorm gelijk moeten zijn aan 'De hooggeboren vrouwe'

Reply:
klopt, een gravin is hooggeboren vrouwe